bedrock/firefox/templates/firefox/nothing-personal/index.html

```diff
@@ -0,0 +1,149 @@
+        <div class="c-browser-window-top">
+          <!-- Content is kept so it matches the size of the visible container above -->
+          <h2>We look forward to not getting to know you</h2>
+        </div>
+        {% endcall %}
+
+        {% call browser_border(class='c-browser-window-top-wrapper c-hidden-browser darker-yellow mzp-l-content mzp-t-content-md') %}
+        <div class="c-browser-window-top">
+          <!-- Content is kept so it matches the size of the visible container above -->
+          <h2>We look forward to not getting to know you</h2>
+        </div>
+        {% endcall %}
+      </div>
+
+        {% call browser_border(class='mzp-l-content mzp-t-content-md mzp-t-content-nospace', heading='Nothing Personal') %}
+          <div class="c-browser-window-top">
+            <h4>You’re great — probably.</h4>
+            <p>We just don’t need a lot of your data to be a <strong>fast, reliable</strong> browser that keeps up with the big guys.</p>
+          </div>
+        {% endcall %}
+
+        <div class="c-thug-life-gif-wrapper mzp-l-content mzp-t-content-md">
+          <div class="c-thug-life-gif c-detached-gif">
+            <img loading="lazy" src="{{ static('img/firefox/nothing-personal/thug-life-fox.gif') }}" alt="">
+          </div>
+        </div>
+
+        <div class="sticky-window">
+          <div class="c-thug-life-gif c-attached-gif">
+            <img loading="lazy" src="{{ static('img/firefox/nothing-personal/thug-life-fox.gif') }}" alt="">
+          </div>
+
+         {% call browser_border(class='mzp-l-content mzp-t-content-md', heading='SFAQs', aria_label='Somewhat Frequently Asked Questions') %}
+          <div class="c-browser-window-top">
+            <h4><span aria-hidden="true">[</span>Somewhat Frequently Asked Questions<span aria-hidden="true">]</span></h4>
+
+            <h5>Are you even any good?</h5>
+            <p>Calm down. And yes. <a href="{{ url('firefox.features.fast') }}">We’re so fast</a> you won’t even notice the difference.</p>
+
+            <h5 aria-label="What makes you so special?">What makes you soooo special?</h5>
+            <p><abbr title="No big deal">NBD</abbr>, but people dump their other browsers <a href="{{ url('firefox.features.index') }}">for our features</a> (like <a href="{{ url('firefox.features.picture-in-picture') }}">popping out videos</a> and a <a href="{{ url('firefox.features.pdf-editor') }}">built-in PDF editor</a>).</p>
+
+            <h5>What does any of this have to do <br> with me?</h5>
+            <p>Hey, you can customize Firefox to be all about you with <a href="{{ url('firefox.features.add-ons') }}">add-ons</a> and <a href="{{ url('firefox.features.customize') }}">themes</a>. Happy?</p>
+
+            <h5>Ok, what’s the privacy-catch?</h5>
+            <p><a href="{{ url('firefox.privacy.index') }}"">We don’t keep track of where you go</a> because 1. Yawn, and 2. We really don’t need it.</p>
```

Comment:
**nit:** remove extra `"`
```suggestion
            <p><a href="{{ url('firefox.privacy.index') }}">We don’t keep track of where you go</a> because 1. Yawn, and 2. We really don’t need it.</p>
```